data/json/recipes/recipe_armor.json

```diff
@@ -7813,6 +7813,94 @@
      [ "tanned_hide", 3 ]
     ]
     ]
+  },{
+    "type" : "recipe",
+    "result": "armor_lightplate",
+    "id_suffix": "assembly",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_SUIT",
+    "skill_used": "tailor",
+    "difficulty": 3,
+    "time": 20000,
+    "reversible": false,
+    "autolearn": false,
```

Comment:
Only reason for tailoring was due to the pre-existing chainmail assembly recipe using that skill.